src/test/java/org/fundacionjala/coding/franzVidangos/StopGninnipsMySdrowTest.java

```diff
@@ -0,0 +1,31 @@
+package org.fundacionjala.coding.franzVidangos;
+
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+
+public class StopGninnipsMySdrowTest {
+
+    @Test
+    public void spinAllWords() {
+        String expectedOut = "olleH dlrow";
+        String actual = SpinWords.sentenceBuilder("Hello world");
+        System.out.println(actual);
```

Comment:
@FranzVidangosPari Please consider removing all the `System.out.println` statements from this class, its considered a code smell and its recommended not include them.